catalog/app/containers/Bucket/PackageDialog/SelectWorkflow.js

```diff
@@ -56,6 +65,8 @@ export default function SelectWorkflow({ disabled, error, items, onChange, value
         <M.Link href={`${docs}/advanced/workflows`} target="_blank">
           Learn about quality workflows
         </M.Link>
+        , or edit workflows{' '}
+        <WorkflowsConfigLink bucket={bucket}>config</WorkflowsConfigLink>
```

Comment:
feels like making the whole "workflows config" phrase clickable makes more sense

Reply:
sure, thanks